iron/recipient.c

```diff
@@ -209,8 +213,28 @@ iron_remove_recipient(const char * login)
  *  user's entry. That one should always be in the list.
  */
 void
-iron_reset_recipients()
+iron_reset_recipients(void)
 {
     if (num_recipients >= 1) num_recipients = 1;
 }
 
+/**
+ *  List the entries in the registered list.
+ *
+ *  For each of the entries (including the current user), display the login.
+ */
+void
+iron_list_recipients(void)
```

Comment:
The recipients are state for the current session, and the current session is only against one host.